app/src/App.vue

```diff
@@ -1,27 +1,44 @@
 <template>
   <div id="app">
-    <HelloWorld msg="Initial Commit"/>
+    <PageHeader :userInfo="userInfo" :menuItems="menuItems" />
+    <router-view></router-view>
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
-
+import PageHeader from './components/header/Header.vue';
 export default {
   name: 'App',
-  components: {
-    HelloWorld
-  }
+  components:{
+    PageHeader
+  },
+  data() {
+    return {
+      userInfo: {
+        name: 'John Doe',
+        avatar: 'Man.jpg',
+      },
+      menuItems: [
+        { id: 1, route: '/', caption: 'Home' },
+        { id: 2, route: '/about', caption: 'About' },
+        { id: 3, route: '/contact', caption: 'Contact' },
+      ],
+    };
```

Comment:
I think the header won't change based on the data yet, so I don't see the need for props. We have one header in our app and I dont think it will be reused elsewhere. Let's do all the configuration in the header component itself.